deploy/operator/overlays/dev-template/kustomization.yaml

```diff
@@ -12,8 +12,18 @@ images:
   newName: quay.io/${JBS_QUAY_IMAGE}/hacbs-jvm-controller
   newTag: dev
 
-
 patches:
+# Note - see base-development for quoting requirements for
+# JBS_S3_SYNC_ENABLED and the linked kubernete issue.
+- patch: |-
+    - op: add
+      path: /spec/template/spec/containers/0/env
+      value:
+        - name: S3_SYNC_ENABLED
+          value: ${JBS_S3_SYNC_ENABLED}
```

Comment:
Ideally patching this as either `"${JBS_S3_SYNC_ENABLED}"` or
```
value: |
	${JBS_S3_SYNC_ENABLED}
```
would transparently patch through but in the former case the quotes are stripped (which is likely https://github.com/kubernetes-sigs/kustomize/issues/4845 ) and in the latter case the inline literal scalar is completely removed and replaced by `value: ${JBS_S3_SYNC_ENABLED}` which actually changed the meaning due to it being then coerced to a bool type